src/challenges.js

```diff
@@ -1,49 +1,138 @@
 // Desafio 1
-function compareTrue() {
-  // seu código aqui
+function compareTrue(x, y) {
+  if (typeof x === 'boolean' && typeof y === 'boolean') {
+    return x && y;
+  }
+  return false;
 }
 
 // Desafio 2
-function calcArea() {
-  // seu código aqui
+function calcArea(base, heigth) {
+  return (base * heigth) / 2;
 }
 
 // Desafio 3
-function splitSentence() {
-  // seu código aqui
+function splitSentence(phrase) {
```

Comment:
O  JavaScript tem uma função nativa chamada 'split' que já divide uma string em um array e o parâmetro dessa função pode ser arbitrado, assim talvez ficasse mais direto o código, oq vc acha?
```suggestion
function splitSentence(phrase) {
return phrase.split(' ');
```

Reply:
Eu pensei em criar minhas funções sem o uso das que já são nativas, porém isso foi uma péssima escolha...pois deixou meu código com alta complexidade.